aws/resource_aws_spot_instance_request.go

```diff
@@ -78,11 +78,12 @@ func resourceAwsSpotInstanceRequest() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			}
-			s["instance_interruption_behaviour"] = &schema.Schema{
```

Comment:
Can we please save this breaking change for another PR? It'll need to be handled in the 2.0.0 upgrade when that occurs. In the meantime, we should deprecate the existing attribute (but that change can be made separately).